docs/faq.md

```diff
@@ -282,6 +282,27 @@ For these reasons, it's strongly recommended to not use brew for native library
 [Homebrew]: https://brew.sh/
 [delocate]: https://github.com/matthew-brett/delocate
 
+### Building Rust wheels
+
+If you build Rust wheels, you need to download the Rust compilers in manylinux.
+If you support 32-bit Windows, you need to add this as a potential target. You
+can do this on GitHub Actions, for example, with:
+
+```yaml
+CIBW_BEFORE_ALL_LINUX: curl -sSf https://sh.rustup.rs | sh -s -- -y
+CIBW_BEFORE_ALL_WINDOWS: rustup target add i686-pc-windows-msvc
+CIBW_ENVIRONMENT_LINUX: "PATH=$HOME/.cargo/bin:$PATH"
+```
+
+You probably need to skip PyPy (if using PyO3, anyway), and Rust does not provide Cargo for musllinux 32-bit:
+
+```toml
+[tool.cibuildwheel]
+skip = ["pp*", "*-musllinux_i686"]
```

Comment:
```suggestion
Rust does not provide Cargo for musllinux 32-bit, so that needs to be skipped:

```toml
[tool.cibuildwheel]
skip = ["*-musllinux_i686"]
```

(Not sure if there's an issue or if it was just me)